common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -220,6 +221,22 @@ def get(request):
     return pipeline_data
 
 
+def get_idp_logout_url_from_running_pipeline(request):
+    """
+    Returns: IdP's logout url associated with running pipeline
+    """
+    if third_party_auth.is_enabled():
+        running_pipeline = get(request)
+        if running_pipeline:
+            tpa_provider = provider.Registry.get_from_pipeline(running_pipeline)
+            if tpa_provider:
+                try:
+                    logout_url = tpa_provider.get_setting('logout_url')
```

Comment:
#NIT why not just do `return tpa_provider.get_setting('logout_url')` so we don't need the extra variable `logout_url`?

Reply:
removed OTU variable.